### ISSUE TITLE
Lighter theme for inline tab-bar

### ISSUE THREAD

**Issue by lloydhumphreys** [Tradeshift/tradeshift-ui#80]
It's grey, it's blocky - it works sometimes, when you also want to use the tab bar as the header of a section. But if a section already has a header, it can be quite heavy.

1. This would work well to alleviate that pain point (I would use it in Risk (and potentially DUI), Magnus in PIM)
2. If we supported icons in these tabs, this could be used as part of @nsp918's design for small tabs within cards.

Basically, we'd like to drop the grey background, and put a blue line beneath the active tab. @sampi !

![2016-11-24 at 15 10](https://cloud.githubusercontent.com/assets/1584500/20601323/60026784-b258-11e6-9fbb-6fed70c6fbab.png)
